package.json

```diff
@@ -10,16 +10,22 @@
   "scripts": {
     "start": "yarn workspace lightning-ui-docs start",
     "prepare": "husky install",
-    "release": "git fetch origin && git checkout develop && git pull origin develop && git pull origin release && git push origin develop && git checkout release && git pull origin release && git pull origin develop && git push origin release",
     "lint": "yarn workspaces foreach run lint",
     "lint:fix": "yarn workspaces foreach run lint:fix",
     "test": "yarn workspaces foreach -v run test",
     "test:updateSnapshot": "yarn workspaces foreach -v run test:updateSnapshot",
-    "build-packages": "yarn workspaces foreach --exclude lightning-ui-docs run build",
     "clean:dist": "yarn workspaces foreach exec rm -rf ./dist ./storybook-static ./tmp",
     "clean": "yarn clean:dist && yarn exec rm -rf node_modules && find ./packages -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "postinstall": "node ./scripts/post-install.js",
-    "createComponent": "node bin/create.js"
+    "createComponent": "node bin/create.js",
+    "release": "git fetch origin && git checkout develop && git pull origin develop && git pull origin release && yarn release:quality && yarn release:build && yarn release:semantic-release && yarn release:build-storybook && yarn release:publish-storybook && git push origin develop && git checkout release && git pull origin release && git pull origin develop && git push origin release",
```

Comment:
Just want to call out the change here that this would be bumping the version in develop, then merging into release as opposed to bumping in release and merging back into develop. I think this is fine, just something to note for the future when we can hopefully have a pipeline stood back up.